### PR TITLE
Feature/base app

### PR DESCRIPTION
**Pull request by MarcoHitsch**
Implemented base structure of the project

### CODE REVIEW

File: Service/Service.API/Controllers/HomeController.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+namespace Tournament40.Service.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class HomeController : ControllerBase
+    {
+        [HttpGet]
+        public string Get()
+        {
+            return "Hello Malte";
```
Comment (mc4t) [inline Service/Service.API/Controllers/HomeController.cs:17]: what is dis? are u kidding me?

File: Service/Service.API/Program.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.UseStartup<Startup>();
```
Comment (mc4t) [inline Service/Service.API/Program.cs:17]: is building web? are you sure?

**Review by mc4t — CHANGES_REQUESTED** [review]

File: Service/Service.API/Controllers/HomeController.cs (hunk 1 above)
Comment (MarcoHitsch) [reply Service/Service.API/Controllers/HomeController.cs:17]: Is your name :P

File: Service/Service.API/Program.cs (hunk 2 above)
Comment (MarcoHitsch) [reply Service/Service.API/Program.cs:17]: Yes im sure

File: Service/Service.API/Controllers/HomeController.cs (hunk 1 above)
Comment (mc4t) [reply Service/Service.API/Controllers/HomeController.cs:17]: is okeh

**Review by mc4t — APPROVED** [review]
is okeh

**Pull request merged**